src/test/java/com/easyrest/tests/BaseTest.java

```diff
@@ -1,22 +1,23 @@
 package com.easyrest.tests;
 
+import com.aventstack.extentreports.ExtentReports;
+import com.aventstack.extentreports.ExtentTest;
+import com.aventstack.extentreports.reporter.ExtentSparkReporter;
 import com.easyrest.constants.Constants;
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.apache.commons.io.FileUtils;
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.testng.annotations.AfterTest;
-import org.testng.annotations.BeforeTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.io.File;
 import java.io.IOException;
 import java.time.Duration;
-import com.aventstack.extentreports.ExtentReports;
```

Comment:
Are you sure that by deleting all this import this you are not breaking anything?

Reply:
It was code formatting, these lines moved up.